multinode-demo/start_nodes.sh

```diff
@@ -41,25 +40,45 @@ leader=
 for ip_addr in "${ip_addr_array[@]}"; do
   echo "$ip_addr"
 
+  ssh-keygen -R "$ip_addr"
+  ssh-keyscan "$ip_addr" >>~/.ssh/known_hosts
+
+  ssh "$remote_user"@"$ip_addr" 'mkdir ~/.ssh'
+
+  ssh -n -f "$remote_user"@"$ip_addr" "sudo service sshguard stop"
+  ssh -n -f "$remote_user"@"$ip_addr" 'sudo apt-get --assume-yes install rsync'
+
+  if [[ -n "$leader" ]]; then
```

Comment:
@rob-solana is gonna tell you to remove those double quotes, `[[` doesn't need them :)